apis/Google.Cloud.Logging.NLog/docs/index.md

```diff
@@ -0,0 +1,36 @@
+{{title}}
+
+`Google.Cloud.Logging.NLog` is a .NET client library to integrate  [Google Stackdriver
+Logging](https://cloud.google.com/logging/) with
+[NLog](http://nlog-project.org/).
+
+{{installation}}
+
+Install the `Google.Cloud.Logging.NLog` package from NuGet. Add it to
```

Comment:
Remove this paragraph - it's generated by the `{{installation}}` part.

Reply:
Done. Also removed from log4net